run_test.sh

```diff
@@ -16,4 +50,18 @@ if [ ! -z "$isFail" ]; then
   exit 1
 fi
 
+# Cleanup
+# Remove the instrumented files
+rm lib/*.in.js 
+rm -r lib/instrument/
+
+sed -i "s:$current_dir:replace_with_path:g" test.json
```

Comment:
This would be more maintainable to replace the `sed` call with a `git checkout test.json`